Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -1260,6 +1261,53 @@ public AndConstraint<TAssertions> NotContainAny(params string[] values)
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    /// <summary>
+    /// Asserts that a string does not contain the specified <paramref name="unexpected"/> string,
+    /// including any leading or trailing whitespace, with the exception of the casing.
+    /// </summary>
+    /// <param name="unexpected">The string that the subject is not expected to contain.</param>
+    /// <param name="occurrenceConstraint">
+    /// A constraint specifying the amount of times a substring should be present within the test subject.
```

Comment:
🔧 This isn't clear for me. How does the occurrence work in combination with "should not contain equivalent of"?